leaflet-distance-marker.js

```diff
@@ -31,12 +31,33 @@ L.DistanceMarkers = L.LayerGroup.extend({
 		var cssClass = options.cssClass || 'dist-marker';
 
 		var zoomLayers = {};
-		var length = L.GeometryUtil.length(line);
+		// Get line coords as an array
+        	  if (typeof line.getLatLngs == 'function') {
+                	var coords = line.getLatLngs();
```

Comment:
What happens if `line.getLatLngs` is not defined (or not a function)?  It seems in that case `coords` will not be defined and lines 56-57 below will cause error.